index.bs

```diff
@@ -1683,14 +1718,18 @@ Methods</h4>
 			4. In case of failure, queue a task on the <a>control thread</a> to execute the following,
 				and abort these steps:
 
-				1. Reject all promises from <a>pendingResumePromises</a> in order, then clear <a>pendingResumePromises</a>.
+				1. Reject all promises from {{AudioContext/[[pending resume promises]]}} in order, then
+					clear {{AudioContext/[[pending resume promises]]}}. Additionaly, remove those promises
+					from {{BaseAudioContext/[[pending promises]]}}.
 
 				2. Reject <em>promise</em>.
```

Comment:
I think this promise can be rejected twice.  Once because it might be a member of `[[pending resume promises]]` and then once again in this step.  Is that ok?

Reply:
You're right, I've removed step 2, and split the first step, that was in fact multiple steps, in two.